src/script/run-cbt.sh

```diff
@@ -104,7 +104,7 @@ if ! $use_existing; then
            --without-dashboard --memstore \
            -o "memstore_device_bytes=34359738368" \
            --crimson --nodaemon --redirect-output \
-           --osd-args "--memory 4G"
+           --osd-args "--memory 32G"
```

Comment:
might be a problem for some users (me...). Should we change the default here?

Reply:
good point. i only have 16 gigs mem in my own test bed...

Reply:
Same here.

Reply:
I think I can move the commit outside this pr:-)